src/app/content/highlights/components/Card.tsx

```diff
@@ -109,189 +133,41 @@ const Card = (props: Props) => {
   };
 
   const commonProps = {
-    className: props.className,
-    isFocused: props.isFocused,
-    onBlur: props.blur,
+    className,
+    isFocused,
+    onBlur: blur,
+    onFocus,
+    onHeightChange,
     onRemove,
     ref: element,
   };
 
-  return !editing && style && annotation ? <DisplayNote
-    {...commonProps}
-    style={style}
-    note={annotation}
-    onEdit={() => setEditing(true)}
-  /> : <EditCard
-    {...commonProps}
-    authenticated={!!props.user}
-    loginLink={props.loginLink}
-    highlight={props.highlight}
-    locationFilterId={locationFilterId}
-    pageId={page.id}
-    onCreate={onCreate}
-    onCancel={() => setEditing(false)}
-    onSave={props.save}
-    data={props.data}
-  />;
-};
-
-/*
- * putting overflow hidden on a page wrapper that aligns with the window edge would
- * include the sidebar, which would break position: sticky.
- *
- * avoiding using an overflow hidden to hide cards when there is not enough space
- * means being very explicit about hiding them so they don't create a horizontal
- * scrollbar.
- *
- * in this case that means using extensive knowledge about the container widths,
- * which unfortunately means knowledge of all the sidebar widths and their state
- * too.
- *
- * consider making a helper like `styleWhenSidebarClosed` maybe `styleWhenContentWidth`
- * that has a selector to get the relevant stuff.
- */
-
-const additionalWidthForCard = (cardWidth + cardContentMargin + cardMinWindowMargin) * 2;
-
-const getHighlightOffset = (container: HTMLElement | undefined, highlight: Highlight) => {
-  if (!container || !highlight.range || !highlight.range.getBoundingClientRect) {
-    return;
-  }
-
-  const {top, bottom} = highlight.range.getBoundingClientRect();
-
-  const offsetParent = container.offsetParent && findElementSelfOrParent(container.offsetParent);
-  const parentOffset = offsetParent ? offsetParent.offsetTop : 0;
-  const scrollOffset = assertWindow().scrollY;
-
-  return {
-    bottom: bottom - parentOffset + scrollOffset,
-    top: top - parentOffset + scrollOffset,
-  };
-};
-
-const getHighlightTopOffset = (container: HTMLElement | undefined, highlight: Highlight): number | undefined => {
-  const offset = getHighlightOffset(container, highlight);
-
-  if (offset) {
-    return offset.top;
-  }
-};
-const getHighlightBottomOffset = (container: HTMLElement | undefined, highlight: Highlight): number | undefined => {
-  const offset = getHighlightOffset(container, highlight);
-
-  if (offset) {
-    return offset.bottom;
-  }
+  return <div onClick={handleClickOnCard}>
+    {
+      !editing && style && annotation ? <DisplayNote
+        {...commonProps}
+        style={style}
+        note={annotation}
+        onEdit={() => setEditing(true)}
+      /> : <EditCard
+        {...commonProps}
+        highlight={highlight}
+        authenticated={!!user}
+        loginLink={loginLink}
```

Comment:
if some of this stuff, like loginLink and authenticated are not used elsewhere, you could select them within `EditCard`. this file already has a LOT of props, so anything we can do to help manage that would be nice